otwcode/otwarchive
# AO3-5065 Fix error where posting chapter would not post work

## Issue

https://otwarchive.atlassian.net/browse/AO3-5065

## Purpose

Fixes an error where posting a chapter of an unposted work did not post the work.

## Testing

Steps to reproduce:

1. Log in
2. Post > New Work
3. Fill in required information
4. Press Preview
5. Press Save Without Posting
6. Follow Add Chapter
7. Fill in required information
8. Press Preview
9. Press Post

Both chapters should be posted. The work should show Chapters: 2/2. You should be able to access it.

## Credit

cresenne, she/her
